### ISSUE TITLE
Dynamic linking for `boot.binfmt.emulatedSystems`

### ISSUE THREAD

**Issue by peterwaller-arm** [Mic92/nix-ld#48]
`nix-ld` is nice, thanks for putting it together!

I was wondering if it would be possible to make it work for emulated binaries. The current failure (running on an x86_64-linux box) is:

```
qemu-aarch64: Could not open '/lib/ld-linux-aarch64.so.1': No such file or directory
```

I'd need some way of pulling in the binaries for the libraries on the alternative system and support from nix-ld somewhere. Does it seem possible?

**Comment by Mic92** [comment]
yes. I think the nixos module could be extended to also to take boot.binfmt.emulatedSystems into account.
